### PR TITLE
Allows xenochimeras to choose icon base

### PR DESCRIPTION
**Pull request by Cameron653**
https://i.imgur.com/MpJSHhF.png
https://i.imgur.com/qLpMjXT.png

- Allows Xenochimeras to choose icon base. Tested and modified so all the xenochimera have a custom datum, making changes done via VV to one during a round not affect the rest. This was required to allow changes in icon base.

- Fixes the "shaved" facial hairstyle so all species can use it. Xenochimera with a custom icon base couldn't use it for some reason in testing

- Fixes a major bug in which organ rejection would occur with the species' organs and kill them.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/station/station_special_vr.dm [hunk 1]
```diff
@@ -15,6 +15,7 @@
 	brute_mod = 0.8		//About as tanky to brute as a Unathi. They'll probably snap and go feral when hurt though.
 	burn_mod =  1.15	//As vulnerable to burn as a Tajara.
 	can_fly = 1 //They have wings by default.
+	var/base_species = "Tajaran" //They have the taj sprite.
```
Comment (Screemonster) [inline code/modules/mob/living/carbon/human/species/station/station_special_vr.dm]: ... they have their own sprite. Slightly different head and shading to Tajara.

Comment (Cameron653) [reply code/modules/mob/living/carbon/human/species/station/station_special_vr.dm]: Oh. Hmm. Will change it to "Xenochimera"

Comment (Cameron653) [reply code/modules/mob/living/carbon/human/species/station/station_special_vr.dm]: Resolved. Made it so Xenochimera are able to choose xenochimera as their icon sprite.

**Comment by SpadesNeil** [comment]
@Screemonster you decide if this is good to merge.

**Comment by Screemonster** [comment]
Uh, if the code's good and actually works then yes. From a lore standpoint I'm absolutely happy with this ability.

**Pull request merged**